pkg/vere/main.c

```diff
@@ -628,6 +628,7 @@ _cw_usage(c3_c* bin_c)
     "utilities:\n",
     "  %s cram %.*s              jam state:\n",
     "  %s dock %.*s              copy binary:\n",
+    "  %s eval                      evaluate hoon from stdin:\n",
```

Comment:
Although you've maintained the alphabetic order by placing this here, I think it should probably be at the top. The missing `<pier>` looks like a mistake otherwise.

Reply:
Moved it to the top